Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM continuumio/anaconda3:latest
```

Comment:
In order to have a cleaner environment I would use [python:3](https://hub.docker.com/_/python/) image.